### DISCUSSION TITLE
a handle dispute request for a never-used npm handle

### DISCUSSION THREAD

**Discussion by TheMikeyRoss** [community/community#128410] [npm]
I'm trying to get @hawa for me team UI library kit at https://github.com/sikka-software/hawa (https://hawa.style) so we can split it into different packages such as @hawa/button, @hawa/input, etc but I noticed an empty user already took the handle @hawa https://www.npmjs.com/~hawa

I filled [this form](https://support.github.com/contact/npm-name-disputes) last month but got no answer :(

I also searched the whole internet for a Hans Wager to contact him but to no avail. 

Is there anything I can do to accelerate my case within NPM? 

Any help is appreciated 🌹

**Comment by MylesBorins** [comment]
@TheMikeyRoss there is no other way to speed up the process unless you have a trademark for the term "hawa". Apologies, but we handle requests first come / first serve.
